src/components/IconGrid.jsx

```diff
@@ -3,23 +3,28 @@ import styled from 'styled-components'
 import IconTile from '../components/IconTile'
 
 const StyledIconGrid = styled.div`
-  max-width: 68.75rem;
-  margin: 0 auto;
-  display: grid;
   width: 100vw;
-  grid-template-columns: 1fr 1fr 1fr 1fr;
-  justify-items: center;
-  gap: 1.5rem;
+  max-width: 1432px;
   margin: 0 auto;
+  /* display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr; */
```

Comment:
You definitely don't need to twist my arm to convert displaying to flex. Let's get rid of all these comments. I'm committed to flex.